ui/partials/perf/comparesubtestctrl.html

```diff
@@ -7,8 +7,13 @@
   <div id="subtest-summary" ng-if="!dataLoading && !errors.length">
     <h1>{{subtestTitle}} subtest summary</h1>
     <ul class="list-inline push-information">
-      <li><strong>Base</strong> - <a href="{{originalProject.getRevisionHref(originalRevision)}}">{{originalRevision}}</a> ({{originalProject.name}})</li>
-      <li><strong>New</strong> - <a href="{{newProject.getRevisionHref(newRevision)}}">{{newRevision}}</a> ({{newProject.name}})</li>
+      <li>
+        <strong>Base</strong> -
+        <revision-describe project="originalProject" project-revision="originalRevision" author="originalAuthor" comment="originalComments"></revision-describe>
+      </li>
```

Comment:
Hmm, maybe it's too complicate and unnecessary to pass each parameter to directive. But I don't know how to make it short. Could you give me some advices about it if you know a better way to pass them? @wlach